apps/visual-test/mock-data/form.ts

```diff
@@ -0,0 +1,57 @@
+export const mockUserMessage = {
+  "_iid": "su-64bae57b-1bea-48c9-92cf-11bbd4894339",
+  "channelType": "group",
+  "messageType": "user",
+  "mentionType": "users",
+  "mentionedUsers": [],
+  "mentionedUserIds": [],
+  "metaArrays": [],
+  "extendedMessage": {},
+  "createdAt": 1728278180937,
+  "updatedAt": 0,
+  "channelUrl": "sendbird_group_channel_200173252_e51dc54565dfb9f22e4c000f7b5edb89c7daf64f",
+  "data": "{\"suggested_replies\": [\"Go to Custom response 2\"], \"respond_mesg_id\": 7619141906, \"workflow\": {\"name\": \"mode test workflow\", \"intent_type\": \"exact_question_match\"}}",
+  "customType": "",
+  "parentMessage": null,
+  "silent": false,
+  "isOperatorMessage": false,
+  "threadInfo": null,
+  "reactions": [],
+  "appleCriticalAlertOptions": null,
+  "scheduledInfo": null,
+  "suggestedReplies": null,
+  "myFeedback": null,
+  "myFeedbackStatus": "NOT_APPLICABLE",
+  "_isContinuousMessages": false,
+  "_scheduledStatus": null,
+  "messageId": 7619141941,
+  "parentMessageId": 0,
+  "ogMetaData": null,
+  "reqId": "",
+  "replyToChannel": false,
+  "errorCode": 0,
+  "sender": {
+    "_iid": "su-64bae57b-1bea-48c9-92cf-11bbd4894339",
+    "userId": "onboarding_bot",
+    "nickname": "My first bot",
+    "plainProfileUrl": "https://file-ap-2.sendbird.com/f80c1e5cb1d74af5ad3e1ecc0679648d.gif",
+    "requireAuth": false,
+    "metaData": {},
+    "connectionStatus": "nonavailable",
+    "isActive": true,
+    "lastSeenAt": null,
+    "preferredLanguages": null,
+    "friendDiscoveryKey": null,
+    "friendName": null,
+    "isBlockedByMe": false,
+    "role": "none"
+  },
+  "sendingStatus": "succeeded",
+  "message": "",
+  "messageParams": null,
+  "translations": {},
+  "translationTargetLanguages": [],
+  "messageSurvivalSeconds": -1,
+  "plugins": [],
+  "_poll": null
```

Comment:
These are all random message values that form test does not care. Don't think much about it.